### PR TITLE
A4A: Fixing minor button styles

### PR DESCRIPTION
**Pull request by coder-karen**
## Proposed Changes

* This PR adds two minor style fixes, as follow-ups to the style changes that were brought about by merging this PR: https://github.com/Automattic/wp-calypso/pull/89414#top
* Those fixes include the Boost action buttons on the Boost preview pane in the sites dashboard, and the license action buttons on the license page, within the expanded content sections.

Design specs (sites dashboard): 1UpoqS1KkCtLXVX68TbMSM-fi-6210_94604

## Testing Instructions

For the below testing steps, make sure to run `yarn start-a8c-for-agencies` locally.

* Visit `http://agencies.localhost:3000/purchases/licenses`. Expand the section for a product, and hover over one of the two buttons (view site or Debug site). The background color should be a light grey, not black as it is in trunk.
* Visit `http://agencies.localhost:3000/sites`, and open the preview pane for a site, where the Boost pane shows the 'Upgrade to auto-optimize' and 'Settings' button. The font color on the Settings button is only marginally different - `#021a23` vs black.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)? PR

### CODE REVIEW

File: client/a8c-for-agencies/sections/purchases/licenses/license-details/style.scss [hunk 1]
```diff
@@ -120,6 +120,11 @@
 	font-weight: 500;
 	line-height: 1.375;
 
+	&:hover {
+		background-color: #fafafa;
```
Comment (cleacos) [inline client/a8c-for-agencies/sections/purchases/licenses/license-details/style.scss]: If it already exists... Could we use some of the existing color vars? from the color theme.

File: client/a8c-for-agencies/sections/sites/site-preview-pane/site-preview-pane-content/style.scss [hunk 2]
```diff
@@ -112,4 +112,8 @@
 	.site-expanded-content__card-content-score-title {
 		text-align: left;
 	}
+
+	a.site-expanded-content__card-button {
+		color: #021a23;
```
Comment (cleacos) [inline client/a8c-for-agencies/sections/sites/site-preview-pane/site-preview-pane-content/style.scss]: `--color-surface-backdrop`

File: client/a8c-for-agencies/sections/purchases/licenses/license-details/style.scss [hunk 3]
```diff
@@ -121,7 +121,7 @@
 	line-height: 1.375;
 
 	&:hover {
-		background-color: #fafafa;
+		background-color: var(--studio-gray-0);
```
Comment (cleacos) [inline client/a8c-for-agencies/sections/purchases/licenses/license-details/style.scss]: `--color-accent-0`

File: client/a8c-for-agencies/sections/sites/site-preview-pane/site-preview-pane-content/style.scss [hunk 4]
```diff
@@ -112,4 +112,8 @@
 	.site-expanded-content__card-content-score-title {
 		text-align: left;
 	}
+
+	a.site-expanded-content__card-button {
+		color: var(--studio-automattic-90);
```
Comment (cleacos) [inline client/a8c-for-agencies/sections/sites/site-preview-pane/site-preview-pane-content/style.scss]: `--color-surface-backdrop`

Note that the `--studio-automattic-90` does not exist

Comment (coder-karen) [reply client/a8c-for-agencies/sections/sites/site-preview-pane/site-preview-pane-content/style.scss]: Changed them all, thanks!

File: client/a8c-for-agencies/sections/purchases/licenses/license-details/style.scss [hunk 5]
```diff
@@ -110,7 +110,7 @@
 	&,
 	.license-details__actions a:not(.license-details__assign-button),
 	.license-details__actions button {
-		background-color: #fafafa;
+		background-color: var(--studio-gray-0);
```
Comment (cleacos) [inline client/a8c-for-agencies/sections/purchases/licenses/license-details/style.scss:113]: ^^

**Review by cleacos — APPROVED** [review]
LGTM!

**Pull request merged**